shared/js/dataDriver.js

```diff
@@ -116,6 +116,17 @@ loop.DataDriver = function() {
       this.update("presence", userId, Object.assign({ isHere }, presence));
     }
 
+    /**
+     * Delete the specified tile record.
+     *
+     * @param {String}  tileId   The id to associate with stored tile.
+     */
+    deleteTileRecord(userId) {
+      // XXX akita bug 1276095: Verify userId is a valid firebase id.
+      this.update("tile", userId, {});
```

Comment:
The `{}` isn't needed and later on we can just check if `value` exists instead of `keys` of 0